README.md

```diff
@@ -12,6 +12,8 @@ Back Office Planning System (BOPS)
 
 ## Preflight
 
+Pre-installed dependencies include ImageMagick and Poppler for generating and previewing thumbnails for images and PDFs respectively.
+
```

Comment:
Where did this come from?

Reply:
Oh, I forgot to mention! When I was setting up my laptop from scratch, I discovered/remembered that you can't just pull the repo, bundle install and build, ImageMagick and Poppler already need to be on your system. So I thought as I was updating the README I'd add this for info.